crates/db/store-sled/src/lib.rs

```diff
@@ -44,15 +44,17 @@ pub fn open_sled_backend(
     dbname: &'static str,
     ops_config: SledDbConfig,
 ) -> anyhow::Result<Arc<SledBackend>> {
-    let sled_db = open_sled_database(datadir, dbname)?;
-    SledBackend::new(sled_db, ops_config)
+    let (raw_db, sled_db) = open_sled_database(datadir, dbname)?;
+    SledBackend::new(raw_db, sled_db, ops_config)
         .map_err(|e| anyhow::anyhow!("Failed to initialize sled backend: {}", e))
         .map(Arc::new)
 }
 
 /// Complete Sled backend with all database types
 #[derive(Debug)]
 pub struct SledBackend {
+    /// Raw sled database handle for opening untyped trees (e.g., MMR database)
```

Comment:
I am curious why do we  want untyped trees. The `typed-sled` library's purpose was to have everything typed so that we can have safe db access.